portfolio/src/main/webapp/script.js

```diff
@@ -63,6 +63,15 @@ function christianComments() {
   });
 }
 
+function uploadedImages() {
+  fetch('/image-upload').then(response => response.json()).then((comments) => {
+    const commentListElement = document.getElementById('display-images');
+    comments.forEach((comment) => {
+      commentListElement.appendChild(createMsgElement(comment));
```

Comment:
I think this would result in incorrect behavior since it's trying to render comments, but only has a list of URLs.

Reply:
Sorry, that was before I edited it. I just wanted to save the pr for uploading images.